lib/tests/misc.nix

```diff
@@ -251,6 +251,21 @@ runTests {
     expected = "&quot;test&quot; &apos;test&apos; &lt; &amp; &gt;";
   };
 
+  testMkShellVars = {
+    expr = ''
+      ${mkShellVars {
+        STRing01 = "just a 'string'";
+        _array_ = [ "of" "strings" ];
+        assoc."with some" = "strings";
```

Comment:
Please test whitespace in all places.

Does this support newlines?

Reply:
It does!